buildbot/osuosl/master/config/builders.py

```diff
@@ -3018,53 +3016,6 @@
                         ],
                     )},
 
-    {'name': "bolt-aarch64-ubuntu-clang",
```

Comment:
We've decided not to drop these builders, right?

Reply:
Correct. That is a Stacked PR, using [user-branches](https://llvm.org/docs/GitHub.html#stacked-pull-requests), and I have to rebase it to pick up the latest changes. Once I do that would be gone.